management/virtualmachine/client.go

```diff
@@ -648,8 +648,8 @@ func (self VirtualMachineClient) createServiceCertDeploymentConf(certPath string
 	return certConfig, nil
 }
 
-func (self VirtualMachineClient) createSshConfig(certPath, userName string) (SSH, error) {
-	sshConfig := SSH{}
+func (self VirtualMachineClient) createSshConfig(certPath, userName string) (*SSH, error) {
```

Comment:
Ok

Sent from my Windows Phone
________________________________
From: Ahmet Alp Balkan
Sent: 3/14/2015 2:38 PM
To: MSOpenTech/azure-sdk-for-go
Cc: Paul Meyer
Subject: Re: [azure-sdk-for-go] Update virtual machine entities to latest spec (#64)


In management/virtualmachine/client.go<https://github.com/MSOpenTech/azure-sdk-for-go/pull/64#discussion_r26442007>:

> @@ -648,8 +648,8 @@ func (self VirtualMachineClient) createServiceCertDeploymentConf(certPath string
>       return certConfig, nil
>  }
>
> -func (self VirtualMachineClient) createSshConfig(certPath, userName string) (SSH, error) {
> -     sshConfig := SSH{}
> +func (self VirtualMachineClient) createSshConfig(certPath, userName string) (*SSH, error) {


just looked at the link. please do introduce the extra var. :)

-
Reply to this email directly or view it on GitHub<https://github.com/MSOpenTech/azure-sdk-for-go/pull/64/files#r26442007>.